modules/system/classes/UpdateManager.php

```diff
@@ -778,15 +775,14 @@ public function requestChangelog()
         if ($result->code != 200) {
             throw new ApplicationException(
                 strlen($result->body)
-                ? $result->body
-                : Lang::get('system::lang.server.response_empty')
+                    ? $result->body
+                    : Lang::get('system::lang.server.response_empty')
```

Comment:
why has this been indented?

Reply:
Phpstorm that ident this way, I usually run auto indent it, follow the PSR.

Reply:
Revert it please.